### ISSUE TITLE
Meaningful Lumi Section Range Includes Invalid BPIX

### ISSUE THREAD

**Issue by jrorie** [cms-l1-dpg/L1PFE_OMS#1]
Hello, 

I noticed on run 320500 that the "Bpix Ready" would sometimes show a red X within the LS range that was stated and physically meaningful. It reports that LS 127-478 are meaningful, but Bpix is not ready for 148-149, 159, 169, 180, 190, 200, 210, 231, 241-242, 252, 263, 274... and more. 

I've included the log file generated as well as the PFE.json file from lxplus as "PFE.txt". This file only has run 320500, but I saw this during my shift with a wide range of runs as well. I've also included the relevant section of the run log. 
[PFE.txt](https://github.com/cms-l1-dpg/L1PFE_OMS/files/2281505/PFE.txt)

Please let me know if there is anything else I can provide.

Thanks, 

Jamal

~~~~ 
## Run 320500 (Collisions18, fill 6998) -- L1T GOOD

Detector components: CTPPS STANDBY, CASTOR EXCLUDED, 
Physically meaningful LS range: [127, 478]
L1 key: l1_trg_collisions2018/v49

L1A Physics rate: <++>kHz
Average PU: <++>

Rates as a function of pileup:
- L1_SingleMu22                  : <++>
- L1_SingleJet180                : <++>
- L1_ETMHF120                    : <++>
- L1_HTT360er                    : <++>
- L1_SingleEG40er2p5             : <++>
- L1_SingleIsoEG32er2p5          : <++>
- L1_DoubleIsoTau32er2p1         : <++>


L1T DQM: <++>
L1TEMU DQM: <++>
~~~~

**Comment by zhenbinwu** [comment]
Thanks Jamal! 

Indeed, the BPix and FPix flag weren't checked in the script. I added it now. 

For the range, the PFE.json does store details lumi section "LSs" and the outer bound "LS"
of these LSs. For the WBM link, it would be easier to handle the big range LS.

**Comment by zhenbinwu** [comment]
Ok, this is now fixed by b9cc86e0dfc961fff633363b83e93abd1421607d and a5ef312261171918503fc13d70b469d20caf2e39

**Comment by jrorie** [comment]
Awesome, thank you very much!
